Assets/Tests/PlayMode/Inventory/InventoryTest.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Aloha.Test
+{
+    public class InventoryTest
+    {
+        [UnityTest]
+        public IEnumerator InventaireTestIterator()
+        {
+
+            // Declaration of an hero
+
+            HeroStats stats = ScriptableObject.CreateInstance<HeroStats>();
+
+            stats.attack = 100;
+            stats.defense = 0;
+            stats.maxHealth = 100;
+
+            GameObject manager = MonoBehaviour.Instantiate(Resources.Load<GameObject>("Prefabs/GameManager"));
+            HeroInstantier.Instance.InstantiateHero(HeroType.Generic);
+            Hero hero = GameManager.Instance.GetHero();
+
+            hero.Init(stats);
+
+            Assert.AreEqual(100, hero.GetStats().maxHealth);
+
+            hero.currentHealth = hero.GetStats().maxHealth;
+
+            Assert.AreEqual(100, hero.currentHealth);
+
+            hero.TakeDamage(20);
+
+            yield return null;
+
+            Assert.AreEqual(80, hero.currentHealth);
```

Comment:
Sur tout le groupe ici, je pense que tu peux regrouper et ne pas mettre de saut de ligne et ajouter un petit commentaire du genre "Check instantiation of the Hero".

Et ne pas oublier de mettre un commentaire à chaque "groupe" de lignes